### DISCUSSION TITLE
[V5] Helpers class

### DISCUSSION THREAD

**Discussion by skrenek** [adonisjs/core#2336] [Help] [answered]
Is there a v5 equivalent for the Helpers class in Adonis 4?  If not, is it on the road map at all?

**Comment by RomainLanz** [comment, marked as answer]
Hey @skrenek! 👋 

You can use the `Application` class for that.

```ts
import Application from '@ioc:Adonis/Core/Application'

Application.configPath(...)
```

**Reply by skrenek** [reply]
Thank you! 🎉
